eclipse/plugins/org.switchyard.tools.ui/src/org/switchyard/tools/ui/common/SwitchYardSettingsGroup.java

```diff
@@ -591,6 +591,9 @@ private void handleRuntimeSelected() {
             final ArtifactVersion version = getRuntimeVersion((IRuntimeComponent) selection.getFirstElement());
             if (version != null) {
                 _runtimeVersionsList.setSelection(new StructuredSelection(version));
+                
+                // to address SWITCHYARD-2968, force the text for Mac OS/X or doesn't get set on selection
```

Comment:
This issue only appears on Mac OS/X - does not appear on Windows or Linux, so the line comment still applies.